docker-32/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM conda/centos6-32-minimal-base:6.9
+MAINTAINER Michael Sarahan <msarahan@continuum.io>
+
+# locales
+RUN yum reinstall -y glibc-common \
+   && yum clean all
+
+# Set the locale
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN yum install -y \
+  bzip2 \
+  libX11 \
+  libXau \
+  libXcb \
+  libXdmcp \
+  libXext \
+  libXrender \
+  libXt \
+  mesa-libGL \
+  mesa-libGLU \
+  openssh-clients \
+  patch \
+  rsync \
+  util-linux \
+  which \
+  wget \
+  && yum clean all
+
+WORKDIR /build_scripts
+COPY install_miniconda.sh /build_scripts
+COPY Miniconda3-* /build_scripts/
```

Comment:
We obviously don't want the Miniconda.sh file checked into this repo.  Let's curl it into the image from https://repo.continuum.io/pkgs/misc/preview/